integrations.json

```diff
@@ -256,6 +256,386 @@
       }
     ]
   },
+  {
+    "name": "AwsSdk",
+    "method_replacements": [
+      {
+        "caller": {},
+        "target": {
+          "assembly": "AWSSDK.SQS",
+          "type": "Amazon.SQS.AmazonSQSClient",
+          "method": "CreateQueueAsync",
+          "signature_types": [
+            "System.Threading.Tasks.Task`1<Amazon.SQS.Model.CreateQueueResponse>",
+            "Amazon.SQS.Model.CreateQueueRequest",
+            "System.Threading.CancellationToken"
+          ],
+          "minimum_major": 3,
+          "minimum_minor": 0,
+          "minimum_patch": 0,
+          "maximum_major": 3,
+          "maximum_minor": 65535,
+          "maximum_patch": 65535
+        },
+        "wrapper": {
+          "assembly": "Datadog.Trace.ClrProfiler.Managed, Version=1.26.3.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
```

Comment:
We are up to 1.27.0 now. Can you regenerate this file?

Reply:
Updated in the latest commit 860bf31684800742947bd053a691335739097718